README.md

```diff
@@ -327,6 +327,21 @@ ecs-cli up --cluster myCluster --empty
 
 This is equivalent to the [create-cluster command](https://docs.aws.amazon.com/cli/latest/reference/ecs/create-cluster.html), and will not create a CloudFormation stack associated with your cluster.
 
+#### User Data
+
+For the EC2 launch type, the ECS CLI always creates EC2 instances that include the following User Data:
+
+```
+#!/bin/bash
+echo ECS_CLUSTER={ clusterName } >> /etc/ecs/ecs.config
+```
+
+This user data directs the EC2 instance to join your ECS Cluster. You can optionally include extra user data with `--extra-user-data`; this flag takes a file name as its argument- it can be used multiple times to specify multiple files. Extra user data can be shell scripts or cloud-init directives- see the EC2 documentation for more information. The ECS CLI takes all the User Data, and packs it into a MIME Multipart archive which can be used by cloud-init on the EC2 instance. The ECS CLI even allows existing MIME Multipart archives to be passed in with `--extra-user-data`; the CLI will unpack the existing archive, and then repack it into the final archive (preserving all header and content type information). Here is an example of specifying extra user data:
```

Comment:
Would you mind wrapping this at 80 or 120 characters?  This is a super long line right now.

It'd also be good to get @nrdlngr to review the documentation here.

Some specific comments since I can't really comment on a part of a single long line...

* You have some sentences with dashes and semicolons.  Can you simplify these sentences so you don't need them (or at least so you don't mix them)?
* Can you link to the relevant section of the EC2 user guide?